src/utils/so/JsonIo.cc

```diff
@@ -85,7 +85,7 @@ namespace
                     break;
                 default: {
                     // unsigned int representation
-                    std::uint8_t c = byte < 0x80 ? byte : byte + 0x80;
+                    std::uint8_t c = static_cast<std::uint8_t>(byte);
```

Comment:
Why this?

Reply:
It's not compiling on mac without the changes here.